internal/hardware/validator.go

```diff
@@ -378,6 +384,10 @@ func (v *validator) getOCPRequirementsForVersion(openshiftVersion string) (*mode
 	return v.VersionedRequirements.GetVersionedHostRequirements(openshiftVersion)
 }
 
+func (v *validator) getHdDeviceTypes() []string {
```

Comment:
What is "Hd"?

Reply:
installable drive like Hard Drive

Reply:
not only installation but a drive that can be use for storage for example CDROM isn't suite for that

Reply:
Hard drive means HDD but this function returns also SSD as type

Reply:
> that can be use for storage

Please explain this in the function documentation, and pick a name that reflects that